Rakefile

```diff
@@ -92,6 +96,14 @@ class RepositoryListUpdater
   def repository_url(fqdn, owner, repository)
     "#{owner_url(fqdn, owner)}#{h(repository)}/"
   end
+
+  def mailto_href_subscribe(to)
+    "mailto:#{h(to)}?cc=null@commit-email.info&amp;subject=Subscribe&amp;body=subscribe"
```

Comment:
Ah, we should call `h` when we use an URL not generate an URL. `&amp;` is for HTML not for URL.
I've updated the current implementation. Please follow the convention.

Reply:
Thank you for the advice.
We should enclose the whole URL in `h` and leave the conversion to `&amp;` to `h`, right?

```rb
h("mailto:#{to}?cc=null@commit-email.info&subject=Subscribe&body=subscribe")
```

Reply:
Oh, I should refer to this fix. Thank you.
https://github.com/kou/commit-email.info/commit/38c11ffc8f244e921c52a1ad0320693094210294

Reply:
This method should not call `h`.
See also: https://github.com/kou/commit-email.info/commit/38c11ffc8f244e921c52a1ad0320693094210294

Reply:
Thank you. I understand.
`h` must be called on href directly.
I have fixed this.